src/main/java/org/akaza/openclinica/service/PdfServiceImpl.java

```diff
@@ -181,7 +182,32 @@ public int addHeaderOrFooter(PDDocument document, String headerMsg,String footer
     }
 
 
-
+   public String preparePdfHeader(Study study, Study site, String studySubjectIdentifier) {
+	  
+	    String siteName = null;
+	    String studyName = null;
+	    String participantID = studySubjectIdentifier.trim();
+	    		   
+	    MessageFormat pdfHeaderFormat1 =  new MessageFormat("{0}: {1} - Participant {2}");
```

Comment:
Make these variables static final, so you don't have to initialize them every time

Reply:
done